exercises/pascals-triangle/canonical-data.json

```diff
@@ -66,7 +66,14 @@
       "input": {
         "count": 6
       },
-      "expected": [[1], [1, 1], [1, 2, 1], [1, 3, 3, 1], [1, 4, 6, 4, 1], [1, 5, 10, 10, 5, 1]]
+      "expected": [
```

Comment:
Is it possible to also have the other expected values formatted with one element per line? That way one really gets a visual indication of the triangle shape

Reply:
with `format-array`, very possible! I will add it to the list of things to be done after this PR. It will be as simple as having `expected` be `multi_line`.